src/platform/windows/window.rs

```diff
@@ -239,6 +239,33 @@ impl Window {
             }
         }
     }
+    
+    /// See the docs in the crate root file.
+    #[inline]
+    pub fn set_resizable(&self, resizable: bool) {
+        if let Ok(mut window_state) = self.window_state.lock() {
+            if window_state.attributes.resizable == resizable {
+                return;
+            }
+            let window = self.window.clone();
+            let mut style = unsafe {
+                winuser::GetWindowLongW(self.window.0, winuser::GWL_STYLE)
+            };
+            if resizable {
+                style |= winuser::WS_SIZEBOX as i32;
+            } else {
+                style &= !winuser::WS_SIZEBOX as i32;
```

Comment:
What's the precedence here?

Reply:
The precedence for removing the `WS_SIZEBOX` when setting resizable to false? I guess I'm not 100% sure what you are referring to.

Reply:
Oh lol. The precedence is `LONG`.

Reply:
Yeah, I just wanted to make sure it was zero-extending *before* flipping the bits. Or rather, I wanted to make sure you thought about that.